docs/_templates/facility_page.rst

```diff
@@ -0,0 +1,46 @@
+Filters of Observation Facility: {{ facility }}
+====================================================
+
+{% set facility_data=data[data['Obs. Facility']==facility] %}
+{% set inst_list=facility_data['Instrument'].unique().tolist() %}
+{% for inst in inst_list %}
+
+{% if inst!='NA' %}
+{% if loop.first %}
+Following **Instruments** are present:
+
+{% endif %}
+
+{{ inst }}
+--------------------------
+
+{% endif %}
+
+{% set inst_data=facility_data[facility_data['Instrument']==inst] %}
+
+.. list-table::
+   :header-rows: 1
+
+   * - {{ inst_data.index.name }}
+{% for col_name in inst_data.columns.values %}
+     - {{ col_name }}
+{% endfor %}
+
+{% for filter_info in inst_data.itertuples() %}
+   * - {{ filter_info[0] }}
```

Comment:
why 10 exactly - that should be looped as well 

Reply:
Because there are 11 columns in our dataframe (0 to 10).

I understand this is an uglier approach, but I chose this to avoid nested loop & save time. Thus right now, I just tried both approach (this & with loop) and compared there execution time and I'm amazed to know this saves only 2s! :open_mouth: 

Hence, as you suggest, I'm resorting to nested loop approach :sweat_smile: